client/flutter/lib/pages/home_page.dart

```diff
@@ -182,40 +182,86 @@ class _HomePageState extends State<HomePage> {
                       onPressed: () {
                         _logAnalyticsEvent('Donate');
                         launch(S.of(context).homePagePageSliverListDonateUrl);
-                    })
-              ),
-              Divider(),
-
-              ListTile(
-                leading: Icon(Icons.share, color: Color(0xffCA6B35)),
-                title: Text(S.of(context).homePagePageSliverListShareTheApp, style: TextStyle(color: Color(0xffCA6B35), fontWeight: FontWeight.w600, fontSize: 20),),
-                trailing: Icon(Icons.arrow_forward_ios),
-                onTap: () {
-                  analytics.logShare(
-                      contentType: 'App', itemId: null, method: 'Website link');
-                  Share.share(
-                      S.of(context).commonWhoAppShareIconButtonDescription);
-                },
+                      })),
+              Divider(height: 1),
+              Material(
+                color: Colors.white,
+                child: ListTile(
+                  contentPadding: EdgeInsets.symmetric(
+                    horizontal: 16,
+                    vertical: 8,
+                  ),
+                  leading: Icon(Icons.share, color: Color(0xffCA6B35)),
+                  title: Text(
+                    S.of(context).homePagePageSliverListShareTheApp,
+                    style: TextStyle(
+                      color: Color(0xffCA6B35),
+                      fontWeight: FontWeight.w600,
+                      fontSize: 20,
+                    ),
+                  ),
+                  trailing: Icon(
+                    Icons.arrow_forward_ios,
+                    color: Color(0xFFC9CDD6),
+                  ),
+                  onTap: () {
+                    analytics.logShare(
+                        contentType: 'App',
+                        itemId: null,
+                        method: 'Website link');
+                    Share.share(
+                        S.of(context).commonWhoAppShareIconButtonDescription);
+                  },
+                ),
               ),
-              Divider(),
-              ListTile(
-                leading: Icon(Icons.settings, color: Color(0xffCA6B35)),
-                title: Text(S.of(context).homePagePageSliverListSettings, style: TextStyle(color: Color(0xffCA6B35), fontWeight: FontWeight.w600, fontSize: 20),),
-                trailing: Icon(Icons.arrow_forward_ios),
-                onTap: () => Navigator.of(context)
-                    .push(MaterialPageRoute(builder: (c) => SettingsPage())),
+              Divider(height: 1),
```

Comment:
For making the divider thicker you need to add thickness:1. Height only defines the space height but not the width of the line.

Reply:
My intention was to remove the extra padding around the divider not change the thickness.

Reply:
Then I think height: 0 would better as the thickness is smaller than 1 by default. But it is okey like this 👍

Reply:
I've had inconsistent behavior with height: 0 in other apps but never mind.

Reply:
Nice to know, thank you